core/src/main/java/hudson/diagnosis/OldDataMonitor.java

```diff
@@ -467,4 +467,9 @@ public Permission getRequiredPermission() {
             return Jenkins.CONFIGURE;
         }
     }
+
+    @Override
```

Comment:
I thought we where leaving AdministrativeMonitors for Admins only?

Guess I was mistaken?

Reply:
We were, it is in the JEP en too

Reply:
It's not the AdministrativeMonitor, it's the "Manage Old Data" menu item, on the JEP it's shown with CONFIGURE permission.